### PR TITLE
120 Collapsible Card

### PR DESCRIPTION
**Pull request by mindfulme**
#### What's this PR do?
Adds collapsible prop with appropriate behavior to the Card.vue component
#### Where should the reviewer start?
at Card.vue
#### How should this be manually tested?
On the dashboard page
#### Any background context you want to provide?
The prop enables the collapsible option,  also user can interact with the collapse button thus show the full card & rotate the arrow button to -90deg
#### What are the relevant tickets?
#120 
#### Screenshots (if appropriate)
screen shot 2018-11-10 at 18 09 57
screen shot 2018-11-10 at 18 09 35
screen shot 2018-11-10 at 18 09 21
screen shot 2018-11-10 at 18 40 50
screen shot 2018-11-11 at 01 24 59

#### URL
Provide a link to your deploy from http://staging-ui.trusty.apasia.tech:8080/
http://staging-ui.trusty.apasia.tech:8080/120-collapsible-card
#### Questions:
-

### CODE REVIEW

File: src/components/Card/Card.vue [hunk 1]
```diff
@@ -53,7 +62,10 @@
           class="header"
           name="header"/>
       </div>
-      <div class="card-body">
+      <div
```
Comment (roma219) [inline src/components/Card/Card.vue]: make it fade in and out with `<transition name="fade">`

File: src/components/Card/Card.vue [hunk 2]
```diff
@@ -167,4 +189,17 @@ export default {
...
+  margin: 0 auto 0 0.1875em;
+  opacity: 0.8;
+}
+
+.collapse-btn--active {
+  margin: 0 auto 0 0.1875em;
+}
+
+.card-collapsed {
+  height: 2.8rem;
```
Comment (roma219) [inline src/components/Card/Card.vue:209]: Make the height change animated with css transition

File: src/components/Card/Card.vue [hunk 3]
```diff
@@ -167,4 +189,17 @@ export default {
   overflow: hidden;
 }
 
+.collapse-btn {
+  transform: rotate(-90deg);
```
Comment (roma219) [inline src/components/Card/Card.vue:195]: make this transform animated with css transition

**Review by roma219 — CHANGES_REQUESTED** [review]
Ok, but needs a few changes. Also reverse the arrow position opened-closed and arrow color (see zeplin)

File: src/components/Card/Card.vue (hunk 1 above)
Comment (mindfulme) [reply src/components/Card/Card.vue]: got it

File: src/components/Card/Card.vue (hunk 2 above)
Comment (mindfulme) [reply src/components/Card/Card.vue:209]: got it

File: src/components/Card/Card.vue (hunk 3 above)
Comment (mindfulme) [reply src/components/Card/Card.vue:195]: got it

**Comment by roma219** [comment]
@MindfulMe And also me the arrow  style `cursor: pointer` please

**Comment by mindfulme** [comment]
@roma219 review, merge if ok

**Review by roma219 — APPROVED** [review]

**Pull request merged**